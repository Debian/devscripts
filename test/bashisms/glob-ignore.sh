#!/bin/sh

GLOBIGNORE="run-tests.sh:BASHISM"
echo *.sh | grep -q run-tests.sh || echo meh
