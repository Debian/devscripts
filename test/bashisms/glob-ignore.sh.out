possible bashism in bashisms/glob-ignore.sh line 3 (GLOBIGNORE=):
GLOBIGNORE="run-tests.sh:BASHISM"
